testing/cypress/e2e/backEndTests/user-content-access.cy.js

```diff
@@ -228,12 +228,9 @@ describe('Test User Role Access to Content Moderation', () => {
 
     // Can not delete content
     cy.visit('/admin/content')
-    pageObjects
-      .editDropDown().then(btn => {
-        cy.get(btn[0]).click()
-      })
-    pageObjects
-      .editOpt().should('not.contain', 'delete')
+    const listItems = []
+    cy.get('[class="dropbutton dropbutton--extrasmall dropbutton--multiple"] li').each(($li) => listItems.push($li.text()))
+    cy.wrap(listItems).should('not.contain', 'delete')
```

Comment:
We need to update the text to match case: 'Delete'.
Please remove the `$` from variable name.
Please remove the listItems array and move the `cy.wrap(li)` into the function of the each loop.